web/app.js

```diff
@@ -1556,9 +1549,8 @@ const PDFViewerApplication = {
     const filename =
       this._contentDispositionFilename || getPDFFileNameFromURL(this.url);
 
-    scripting.createSandbox({
+    const success = await scripting.createSandbox({
```

Comment:
`createSandbox` is expected to succeed under normal circumstances. A try-catch or a `.catch` (or equivalently a `.then` handler would be more idiomatic than a single boolean.